app/controllers/admin/users_controller.rb

```diff
@@ -15,9 +15,8 @@ def index
   end
 
   def update
-    if user.update_attributes(user_params)
-      role = Role.find(user_params[:role_id]).name
-      Rails.logger.info("#{current_user.name_and_email} changed the role of #{user.name_and_email} to #{role}")
+    if user.update_attributes(params.require(:user).permit(:role_id))
```

Comment:
I liked having user_params as separate function to keep noise down in this line, but not too bothered by it.